api/jobs.go

```diff
@@ -97,6 +92,9 @@ func (api *API) CreateJobHandler(w http.ResponseWriter, req *http.Request) {
 		}
 	}
 
+	newJob.Links.Self = host + "/v1" + newJob.Links.Self
```

Comment:
If we are storing the version number (let's say `v1`) in the links....in the scenario, when this version becomes deprecated, the links become invalid. I understand that the link might still work as the default is to get the latest (I haven't tested this yet). But does it matter to include the version in the link?